core/src/main/scala/cats/Traverse.scala

```diff
@@ -135,12 +135,24 @@ import simulacrum.typeclass
   override def map[A, B](fa: F[A])(f: A => B): F[B] =
     traverse[Id, A, B](fa)(f)
 
+  /**
+   * Akin to [[map]], but allows to keep track of a state value
+   * when calling the function.
+   */
+  def mapAccumulate[A, S, B](fa: F[A])(init: S)(f: (A, S) => (B, S)): (F[B], S) =
```

Comment:
Why is init in a separate parameter group. Do we usually do that? Why not group with fa?

Reply:
Probably this was to appease simulacrum, which was used to bootstrap this change?

Reply:
Sorry, I am just kind of used to currying by default... I made the same mistake on the `contains_` PR 😅